.gitignore

```diff
@@ -2,3 +2,5 @@
 target/
 dbt_modules/
 logs/
+.DS_Store
+integration_test/.DS_Store
```

Comment:
@fivetran-joemarkiewicz FYI I added .DS_Store to the gitignore. Can you delete these files from the PR please?